Source/Core/Resource.js

```diff
@@ -1235,7 +1318,178 @@ define([
     };
 
     /**
-     * Asynchronously posts data the given resource.  Returns a promise that will resolve to
+     * Asynchronously deletes the given resource.  Returns a promise that will resolve to
+     * the result once loaded, or reject if the resource failed to load.  The data is loaded
+     * using XMLHttpRequest, which means that in order to make requests to another origin,
+     * the server must have Cross-Origin Resource Sharing (CORS) headers enabled.
+     *
+     * @param {Object} [options] Object with the following properties:
+     * @param {String} [options.responseType] The type of response.  This controls the type of item returned.
+     * @param {Object} [options.headers] Additional HTTP headers to send with the request, if any.
+     * @param {String} [options.overrideMimeType] Overrides the MIME type returned by the server.
+     * @returns {Promise.<Object>|undefined} a promise that will resolve to the requested data when loaded. Returns undefined if <code>request.throttle</code> is true and the request does not have high enough priority.
+     *
+     *
+     * @example
+     * // Load a single resource asynchronously. In real code, you should use loadBlob instead.
```

Comment:
This one is definitely a copy and paste error, remove the commend and change `blob` to `body` (I assume DELETE returns the body without interpreting it at all?)

Reply:
Nevermind, I read the doc, the example should definitely specify a response type (probably JSON) so that users see a real world use case.